Sources/WebDriver/Requests.swift

```diff
@@ -531,4 +531,32 @@ public enum Requests {
 
         public typealias Response = WebDriverStatus
     }
+
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidorientation
+    public enum SessionOrientation {
+        public struct Post: Request {
+            public var session: String
+            public var orientation: ScreenOrientation
+
+            public var pathComponents: [String] { ["session", session, "orientation"] }
+            public var method: HTTPMethod {.post}
```

Comment:
Please add spaces within curly braces throughout.
```suggestion
            public var method: HTTPMethod { .post }
```